src/main/java/com/tjtanjin/steve/ui/UiHandler.java

```diff
@@ -67,10 +67,19 @@ public void showInfo(String info) {
      *
      * @param msg error message to print
      */
-    public void showError(String msg) {
+    private void showError(String msg) {
         dialogContainer.getChildren().add(DialogBox.getSteveDialog(msg, STEVE_IMAGE));
     }
 
+    /**
+     * Terminates the program with a final message.
+     *
+     * @param crashMsg message to print before termination
+     */
+    private void showTerminate(String crashMsg) {
```

Comment:
Would this method name be confusing for some readers?

Reply:
On second thought it is indeed unclear as to what it is showing. I updated the other methods as well to state more clearly that a message is being shown. Thank you for highlighting this! :D